### ISSUE TITLE
🐛 Adding Multiple Developer Extension does't work.

### ISSUE THREAD

**Issue by Kidcredo** [manojVivek/responsively-app#236]
As shown in the image below i have Vue.js Dev tools and React Developer tools extension installed. 
It only show React Developer tools which was installed after Vue.js Dev tools.
Vue.js Dev tools has disappeared from the Dev tools.
This means only one developer tools extension work at a time.

![image](https://user-images.githubusercontent.com/15522605/86316889-83d3f100-bc1d-11ea-9bea-edfcbc362e24.png)

**Comment by manojVivek** [comment]
@Kidcredo Are you seeing the Vuejs tab when you delete the `React Developer Tools` extension? 

It seems to work pretty fine for me.
<img width="1229" alt="Screenshot 2020-07-02 at 12 16 29 PM" src="https://user-images.githubusercontent.com/1283424/86325607-efd64980-bc5d-11ea-90f3-d639679ef196.png">

Can you please add the exact steps to reproduce it?

**Comment by Kidcredo** [comment]
Steps to reproduce this issue on the latest release.
1.Install Vue Devtools using the extentsion ID
(Now when its a Vue tab it shows up.)
2. Installed React Dev tools (Now only react tabs show up even if its a vue app.)

When i delete react dev tool the Vue tab is visible again.

**Comment by Kidcredo** [comment]
Ok i understand what happened a bit here is how it happened.

Did a clean install of the latest version of the app.

1. I installed Vuejs devtools 
2. Visited vuejs.org Website to check it and it worked 
3. I closed the devtools.
4. Installed the React Dev tools 
5.Opened Reactjs.org website to see if it works and only react dev tools was visible.
6. I switched to Vuejs.org website without closing the devtools and it was still showing React tabs. No vue tab Visible yet

But here is what i did next.
After closing the Devtools and reopening them now only Vue tabs show up and React dissapeared.
